### DISCUSSION TITLE
Tiptap does not support paste image into base64

### DISCUSSION THREAD

**Discussion by pingf** [mantinedev/mantine#3175] [Feature requests]
### What package has an issue

@mantine/tiptap

### Describe the bug

as described in the title, we want to migrate to the new rte.
it seems it lacks this key feature  which is really nice for simple apps

### What version of @mantine/hooks page do you have in package.json?

5.9.3

### Do you know how to fix the issue

No

### Are you willing to participate in fixing this issue and create a pull request with the fix

None

**Comment by rtivital** [comment]
You can use any extensions that tiptap provides, including images
